src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.replace(/^\n+/g, '').split(';');
+  const data = {};
+
+  styleArray.forEach((elem) => {
+    const style = elem.trim();
+
+    if (!style) {
+      return;
+    }
+
+    const [key, value] = style.split(':').map((item) => item.trim());
```

Comment:
The `split(':')` operation assumes that there is exactly one colon per style declaration. If there are malformed entries or additional colons, this could lead to unexpected behavior. Consider adding error handling or validation for such cases.